bciers/apps/administration/tests/components/contacts/ContactForm.test.tsx

```diff
@@ -258,6 +259,12 @@ describe("ContactForm component", () => {
           "/contacts/123?contacts_title=John Doe",
         );
       });
+
```

Comment:
Suggest adding this assertion to the editing test too. Sidenote, it looks like there are two, `creates a new contact, edits it, and submits the updated contact` and `updates existing contact form data and hits the correct endpoint` and we probably only need one. There's also a `fills the mandatory form fields, creates new contact, and redirects on success`, so maybe it's `creates a new contact, edits it, and submits the updated contact` that's the most duplicated?

Reply:
@BCerki 

I believe `creates a new contact, edits it, and submits the updated contact` is to test that same page creating\editing (without browser refresh) saves a create via POST then saves updates via PUT; so, a slightly different test from the create test and the edit test.